blocks/core/ff_module/ff_module-profile-response-button/ff_module-profile-response-button.md

```diff
@@ -1,9 +1,32 @@
 <div data-ff_module-profile-response-button=""/>
 
-# Here be a profile-response button
+# Profile response button
 
 ## Markup 
 ```
 <div data-ff_module-profile-response-button=""/>
 ```
+### Props
 
+- `onSelect`: Function
+- `uiState`: String; css defined for `is-selected`, `is-updated` (default), `is-reviewed` in Melody
+- `guid`: String; unique id
+- `label`: String; Name of Recipient
+- `status`: String; Response Status
```

Comment:
Does the status text always correspond with `uiState` (ignoring the selected state)? For instance, if resubmission if requested, we always show the same UI state and message? If so, perhaps it's worth just being able to pass a string like `resubmission-requested` in here to indicate the status, and then having the component decide on the message and styling?